kokoro/presubmit.cfg

```diff
@@ -12,4 +12,66 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-build_file: "dataflux-pytorch/kokoro/build.sh"
+build_file: "dataflux-pytorch/kokoro/presubmit.sh"
+
+env_vars {
+  key: "PROJECT"
+  value: "dataflux-project"
+}
+
+env_vars {
+  key: "BUCKET"
+  value: "official-dataflux-tess"
+}
+
+env_vars {
+  key: "PREFIX"
+  value: "UNet3D/micro/100KB-500MB/train"
+}
+
+env_vars {
+  key: "LIST_WORKERS"
+  value: "32"
+}
+
+env_vars {
+  key: "FILE_COUNT"
+  value: "5000"
+}
+
+env_vars {
+  key: "TOTAL_FILE_SIZE"
+  value: "501770000"
+}
+
+env_vars {
+  key: "MAX_COMPOSE_BYTES"
+  value: "100000000"
+}
+
+env_vars {
```

Comment:
Nice to see that we are using the same timeouts as in dataflux-client-python! For listing, it should be mostly equivalent but for "download", using the same timeout will make the check tighter due to the various operations that DataLoader has and the `sleep_time`.

It's okay though. I just wanted to point out this implication.

Reply:
Yes if it exceeds the timeout or demonstrates inconsistency i'll extend it